README.md

```diff
@@ -14,6 +14,7 @@
 - **[Documentation](https://mochajs.org)**
```

Comment:
Trivially, but this will be faster if the link ends with a slash.
`https://mochajs.org/`